src/vsr/replica.zig

```diff
@@ -9021,6 +8971,10 @@ pub fn ReplicaType(
             self.grid.free_set_checkpoint.reset();
             self.client_sessions_checkpoint.reset();
             self.client_sessions.reset();
+            // Faulty bits will be set in sync_content().
+            while (self.client_replies.faulty.findFirstSet()) |slot| {
+                self.client_replies.faulty.unset(slot);
+            }
```

Comment:
This is from sync_requesting_checkpoint_callback! And this _should not_ be dead code, we should clear faulty status of client replies!